lib/deps.edn

```diff
@@ -1,3 +1,4 @@
+;; -*- cider-clojure-cli-aliases: ":test"; -*-
```

Comment:
Instead of adding it to the top of this file, I'd suggest adding this to the `.dir-locals.el`?

Reply:
done.